ocs_ci/ocs/resources/csv.py

```diff
@@ -68,6 +68,9 @@ def wait_for_phase(self, phase, timeout=300, sleep=5):
             timeout (int): Timeout in seconds to wait for desired phase
             sleep (int): Time in seconds to sleep between attempts
 
+        Raises:
+            ResourceInUnexpectedState: In case the
```

Comment:
looks like sentence is not completed, In case the ?

Reply:
done, somehow messed up this PR. Don't know.